### ISSUE TITLE
Oxygen License Key Can Be Simply Retrieve From Dashboard

### ISSUE THREAD

**Issue by syafiyullah** [soflyy/oxygen-bugs-and-features#1768]
Oxygen builder is really an awesome web builder. But it will be great if you can improve the license key authentication method to avoid license key leak to client.

**Problem**
By simply inspect elements on the license field, anyone can simply see the license code or if you change the field type to "txt" you will also able to see the license key. It will be a serious issue if we build a website for client and client want to have full admin access to their website. 

**To Reproduce**
1. Go to Wordpress Dashboard => Oxygen => Settings => License
2. Right click on license field and choose "Inspect Elements"
3. From the inspector details, you will able to see the license key 
4. or, if you chnage the filed type to "txt", you will able to see teh liceseky clearly.

**Expected behavior**
The license key should be encrypted and have to be two way authentication. 

**What actually happens**
The license key can be easily get by anyone who have admin permission to the wordpress

**Screenshots**
![Oxygen](https://user-images.githubusercontent.com/74046523/101647079-49476d80-3a73-11eb-948f-7670fda83976.png)

**Desktop (please complete the following information):**
- OS: Windows
 - Browser : Chrome / Edge / Mozilla
 - Oxygen Version : 3.6.1

Kindly please help to fix this securoty flaws.
Thank you.

**Comment by Uwishvajith** [comment]
Yeah same issue here. I need to stop leaking my licence key to the clients too

**Comment by Warren-WordPress** [comment]
This is interesting. For now, the solution is to delete the Oxygen license immediately after an update. 
For example: On a client site remove the site key and only use it for updates.  After updating remove the site key immediately. The only Admin accessing the site during an update should be the one doing the update. My client pays me to do the updates or they client can purchase their own license key for the site allowing them to do the update.

**Comment by derSebastian** [comment]
The license is also stored in plain text in the database! Same issue apply as the mentioned above. Additionally that means that anyone with access to the DB server (compromised shared host for example) could use that license until the user happens to check the registered sites tag (also NO notifications are sent).

Why not use a token? That would also solve a request for site/domain specific installations per client as seen in issue [1675] (https://github.com/soflyy/oxygen-bugs-and-features/issues/1675) as well. Enter key, authenticate, generate token, save token for future comparison/updates.

**Comment by sunnyliumr** [comment]
I would like to ask when such an obvious and important function can be solved. I hope the official can give a positive answer. Thank you!

**Comment by reeacee** [comment]
I have the same issue. Please fix it in the next updates. Thanks! :-)

**Comment by bearbops** [comment]
It seems that Oxygen doesn't care that our licenses are accessible to anyone. It's not very correct behavior towards those who paid for these licenses.

**Comment by lucilecollignon** [comment]
Any news ?
